### ISSUE TITLE
Can't fly around the world

### ISSUE THREAD

**Issue by SLembas** [mcedit/mcedit2#106]
Unlike in MCEdit 1.X I can't fly around the world. Only can rise by holding Space.

Steps to reproduce:
1. Open any world.
2. Try to fly.

MCEdit 2.0 alpha-417
Minecraft 1.8.6
Windows 7 SP1 64-bit

**Comment by codewarrior0** [comment]
Explain "Try to fly" in terms of mouse and keyboard input.

**Comment by SLembas** [comment]
I always used WASD to fly. W - forward; S - backward; A - strafe left; D - strafe right.

**Comment by codewarrior0** [comment]
Can you turn the camera by holding the right mouse button and moving the mouse?

Have you tried clicking on the "Controls" button to make sure the controls are set to WASD? Did you try to see if you can change the controls to different keys?

**Comment by SLembas** [comment]
Ok, i found a trouble. I just changed keyboard language from Russian to English and it worked, but it isn't normal.

**Comment by SLembas** [comment]
Another question: how to change FoV?

**Comment by codewarrior0** [comment]
It's not possible to change the FoV right now.

Could you do me a favor? Switch your keyboard back to Russian, and try using the Controls button to change the movement keys. Tell me if you can change them to whatever the Russian keys are and if so, let me know if you can move around using those keys.

**Comment by SLembas** [comment]
Yes, I can change controls to Russian. And all works now only with Russian keyboard.

**Comment by T-BoneForest** [comment]
I can't see a "controls" button.  Trying to configure for a Dvorak programmer's keyboard.
